components/ArtPiecePreview/index.js

```diff
@@ -0,0 +1,13 @@
+export default function ArtPiecePreview({ imageSource, title, artist }) {
```

Comment:
stick to the naming you are provided with in the Userstory. image, title, artist